### PR TITLE
Make staff/history modals behave in all browsers

### PR DESCRIPTION
**Pull request by caesar2164**
Before:
screen shot 2017-01-27 at 11 28 57 am

After:
screen shot 2017-01-27 at 11 29 04 am

Offending styling:
screen shot 2017-01-30 at 3 25 59 pm

### CODE REVIEW

**Comment by caesar2164** [comment]
@mondiaz, @caseylitton, & @stvstnfrd

File: static/sass/_lagunita.scss [hunk 1]
```diff
@@ -344,6 +344,18 @@ section.introduction h1.sr {
...
 
+// Modal styling
+#seq_content .vert-mod {
+    section.modal.staff-modal,
+    section.modal.history-modal {
+        left: 0 !important;
+        top: 0 !important;
+        width: 90% !important;
+        height: 85% !important;
+        margin: 5% !important;
```
Comment (stvstnfrd) [inline static/sass/_lagunita.scss]: What is so important

File: static/sass/_lagunita.scss [hunk 2]
```diff
@@ -344,6 +344,18 @@ section.introduction h1.sr {
     min-height: 100px;
 }
 
+// Modal styling
```
Comment (stvstnfrd) [inline static/sass/_lagunita.scss]: Where are the styling and content for this issue coming from?

Would this issue affect any theme running our platform code?
If so, would we want to include the fix in the platform styling?

File: static/sass/_lagunita.scss (hunk 1 above)
Comment (stvstnfrd) [inline static/sass/_lagunita.scss]: Can we do away with importants?

Comment (caesar2164) [inline static/sass/_lagunita.scss]: sadly no, these are competing with JS added element styling, so `!important` is the only way to override it. 😢

**Comment by caesar2164** [comment]
closing in favor of a platform PR: https://github.com/Stanford-Online/edx-platform/pull/578

**Pull request closed without merging**